generators/2.0/java/index.js

```diff
@@ -27,15 +30,15 @@ module.exports = class extends Generator {
                     'Gradle',
                 ]
             },
-            // {
-            //     type: 'list',
-            //     name: 'framework',
-            //     message: 'Select your framework',
-            //     choices: [
-            //         'Java (no framework)',
-            //         'SpringBoot (experimental)'
-            //     ]
-            // },
+            {
+                type: 'list',
+                name: 'framework',
+                message: 'Select your framework',
+                choices: [
+                    'Java (no framework)',
```

Comment:
Could we have constants or an enum for the two frameworks? Strings are duplicated below (l. 92)

Reply:
I'm not sure if we recommend developers to use Gradle or not but I think the developers are still able to create a bot with maven.